src/app/Dashboard/Dashboard.tsx

```diff
@@ -99,89 +100,96 @@ const PlaceholderCard: React.FunctionComponent<
   );
 };
 
-export const DashboardCards: DashboardCardDescriptor[] = [
-  AutomatedAnalysisCardDescriptor,
-  {
-    title: 'None Placeholder',
-    description: 'placeholder',
-    descriptionFull: 'This is a do-nothing placeholder with no config',
-    component: PlaceholderCard,
-    propControls: [],
-  },
-  {
-    title: 'All Placeholder',
-    description: 'placeholder',
-    descriptionFull: 'This is a do-nothing placeholder with all the config',
-    component: PlaceholderCard,
-    propControls: [
-      {
-        name: 'string',
-        key: 'title',
-        defaultValue: 'a short text',
-        description: 'a string input',
-        kind: 'string',
-      },
-      {
-        name: 'text',
-        key: 'message',
-        defaultValue: 'a long text',
-        description: 'a text input',
-        kind: 'text',
-      },
-      {
-        name: 'menu select',
-        key: 'menu',
-        values: ['choices', 'options'],
-        defaultValue: '',
-        description: 'a selection menu',
-        kind: 'select',
-      },
-      {
-        name: 'menu select 2',
-        key: 'asyncmenu',
-        values: new Observable((subscriber) => {
-          let count = 0;
-          const id = setInterval(() => {
-            if (count > 2) {
-              clearInterval(id);
-              setTimeout(() => subscriber.error('Timed Out'), 5000);
-            }
-            subscriber.next(`async ${count++}`);
-          }, 1000);
-        }),
-        defaultValue: '',
-        description: 'an async stream selection menu',
-        kind: 'select',
-      },
+export const getDashboardCards: (featureLevel?: FeatureLevel) => DashboardCardDescriptor[] = (
+  featureLevel?: FeatureLevel
+) => {
+  let cards = [AutomatedAnalysisCardDescriptor];
+  if (featureLevel === undefined || featureLevel === FeatureLevel.DEVELOPMENT) {
```

Comment:
Does `featureLevel === undefined` means defaulting to the lowest level (DEVELOPMENT)?

Then i guess we could clean up with default?

```typescript
export const getDashboardCards: (featureLevel?: FeatureLevel) => DashboardCardDescriptor[] = (
  featureLevel: FeatureLevel = FeatureLevel.DEVELOPMENT
) => {}
```

Reply:
Ah yea, that's a good syntax trick to make it clearer to read.

It doesn't mean lowest level, it really does mean when callers don't pass an argument - like the other two `getConfigByX` functions in that file. In that case we just want the whole list of available configs since we're going to filter them out by a (hopefully) unique property.

Reply:
Ahh okayyy makes sense! I think we are good with either way since its just a syntax trick but I guess `featureLevel === undefined` will be clearer in this sense.